src/App.jsx

```diff
@@ -1,104 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import { PostList } from './components/PostList/PostList';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+const getUserById = postId => {
+  return usersFromServer.find(user => user.id === postId) || null;
+};
+
+const getCommentById = postId => {
+  return (
+    commentsFromServer.filter(comment => comment.postId === postId) || null
```

Comment:
Array/filter always return truthy value
```suggestion
    commentsFromServer.filter(comment => comment.postId === postId)
```